shared/js/blobview.js

```diff
@@ -260,114 +260,26 @@ var BlobView = (function() {
 
     getASCIIText: function(offset, len) {
       var bytes = new Uint8Array(this.buffer, offset + this.viewOffset, len);
-      return String.fromCharCode.apply(String, bytes);
+      var decoder = new TextDecoder('ascii');
+      return decoder.decode(bytes);
```

Comment:
Yes, there is. The ID3 (v1 and v2) parsers expect Latin-1 strings, which is one of the more-common places this functin is used. For any non-ASCII Latin-1 character, we'd want to convert it to the correct Unicode code point so we don't get mojibake.

`TextDecoder` is also about 20% faster (at least on Desktop).